### PR TITLE
Add loadbot documentation

### PR DESCRIPTION
**Pull request by PtitLuca**
This PR includes the instructions for running the loadbot.

### CODE REVIEW

File: docs/how-tos/howto-run-loadbot.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+
+The Polygon SDK binary includes a `loadbot` command, responsible for sending a specific number of transactions per
+second.
+
+This _how-to_ will explain how to run the loadbot on a single node configuration using 2 accounts prefilled with a
+balance of 1000 ETH each.
+
+:::note
+
+This is not the only parameter you can specify when running the loadbot.
```
Comment (zivkovicmilos) [inline docs/how-tos/howto-run-loadbot.md:26]: We can add a link to the actual loadbot command / flags here, in the Commands section

File: docs/how-tos/howto-run-loadbot.md [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+````bash
+go run main.go genesis --premine 0x1010101010101010101010101010101010101010 --premine 0x1010101010101010101010101010101010101020
+````
+
+The **premine** flag sets the address that should be included with a starting balance in the **genesis** file.<br />
+In this case, the addresses `0x1010101010101010101010101010101010101010` and
+`0x1010101010101010101010101010101010101020` will have a starting **default balance** of
+`0x3635C9ADC5DEA00000 wei`.
+
+## Step 2: Start the Polygon SDK in dev mode
```
Comment (zivkovicmilos) [inline docs/how-tos/howto-run-loadbot.md:43]: Does it need to start in dev mode?

**Review by zivkovicmilos — APPROVED** [review]
Looks great to me 💯 

Left a few minor comments to discuss.

Glad we could make this feature public 👍

File: docs/how-tos/howto-run-loadbot.md (hunk 1 above)
Comment (zivkovicmilos) [reply docs/how-tos/howto-run-loadbot.md:26]: Speaking of which, I don't see the command added to the CLI Commands section.

We should update that document as well 👍

File: docs/how-tos/howto-run-loadbot.md (hunk 2 above)
Comment (lazartravica) [reply docs/how-tos/howto-run-loadbot.md:43]: @PtitLuca I don't think that people who want to stress test the network need instructions on generating the genesis file and everything else.

Simply add as requirements to the top of the doc that you should have:
- A working Polygon-SDK network
- JSON-RPC reachable
- gRPC reachable
- Private key of the account with X tokens to use for testing
- anything else I may have missed

**Review by lazartravica — CHANGES_REQUESTED** [review]
@PtitLuca Loadbot is under the startup commands header. Create a new section at the bottom: "Utilities"

**Review by lazartravica — APPROVED** [review]

**Pull request merged**